### PR TITLE
Update KB article on SSH issues

### LINKED ISSUE #1946: Create Knowledge Base Article for issues SSH'ing to multi-process app
See support requests [here](https://gsa-tts.slack.com/archives/C09CR1Q9Z/p1619792107014000) and [here](https://gsa-tts.slack.com/archives/G3JNU3HJL/p1625092616000100) for more background info.

Note, issue may not manifest with `cf ssh` but will with `ssh` or `scp`.

Open question: should this be an update to the existing SSH KB article, or a separate one?

### PR DESCRIPTION
**Pull request by mheadd**
## Changes proposed in this pull request:
- Updated the existing article on SSH connection issues
- Clarified how to use tools like `ssh` and `scp` with apps that on V3 of the CAPI.
- Fixes #1946 

:sunglasses: [PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/mheadd-patch-9/knowledge-base/2021-05-17-troubleshooting-ssh-connections/)

## Security Considerations
None. Content changes only.

### CODE REVIEW

File: _kbarticles/2021-05-17-troubleshooting-ssh-connections.md [hunk 1]
```diff
@@ -19,15 +20,26 @@ Some things to check:
...
+
 ### Changes to your app in the CAPI
 
 If you are still experiencing trouble connecting to your app using SSH, it may be the result of some changes in a newer version of the Cloud Foundry API (CAPI). In prior versions of the CAPI, app instances used a single process. In version 3 of the CAPI, there can be more than 1 process that makes up an app. This may occasionally cause issues when trying to SSH to your app. 
 
 If a modification or update was made to an app using a v3 command or process - even if that change was made by someone else (e.g., another member of a development team) - the app’s schema in the Cloud Controller might have changed to version 3, and a standard cf ssh may no longer work. 
 
-If this occurs, you can try using `cf v3-ssh {app-name}`. By default a cf v3-ssh will select the `web` process. You can also select a different process if needed with v3-ssh using the `--process` flag.
\ No newline at end of file
+If this occurs and you are using v6 of the `cf` CLI, you can try using `cf v3-ssh {app-name}`. By default a cf v3-ssh will select the `web` process. You can also select a different process if needed with v3-ssh using the `--process` flag.
```
Comment (ccostino) [inline _kbarticles/2021-05-17-troubleshooting-ssh-connections.md]: Is it just this part that is distinctive between the V6 and V7 CLIs, or the instructions below as well?  In other words, will the steps that follow this note also work for V7?

If that's the case, highlighting that this statement is for V6 only will help cut down on confusion.  Alternatively, putting the remaining content below under a new sub-heading will also work.

**Review by ccostino — COMMENTED** [review]
Thank you, @mheadd!  Just one note of clarification on the last section as it was a bit confusing what was for just V6 vs. V7.

File: _kbarticles/2021-05-17-troubleshooting-ssh-connections.md (hunk 1 above)
Comment (mheadd) [reply _kbarticles/2021-05-17-troubleshooting-ssh-connections.md]: Made some changes

**Review by ccostino — APPROVED** [review]
Awesome, thank you @mheadd!

**Pull request merged**